src/multivariate.jl

```diff
@@ -44,12 +131,10 @@ struct TuringScalMvNormal{Tm<:AbstractVector, Tσ<:Real} <: ContinuousMultivaria
 end
 
 Base.length(d::TuringScalMvNormal) = length(d.m)
-Base.size(d::TuringScalMvNormal) = (length(d), )
-function Distributions.rand(rng::Random.AbstractRNG, d::TuringScalMvNormal)
-    return d.m .+ d.σ .* randn(rng, length(d))
-end
-function Distributions.rand(rng::Random.AbstractRNG, d::TuringScalMvNormal, n::Int)
-    return d.m .+ d.σ .* randn(rng, length(d), n)
+Base.size(d::TuringScalMvNormal) = (length(d), length(d))
```

Comment:
IMO this is incorrect (and was correct before). `MvNormal` returns vectors and not matrices.

Reply:
```suggestion
Base.size(d::TuringScalMvNormal) = (length(d),)
```